exercises/robot-name/robot-name-test.rkt

```diff
@@ -0,0 +1,38 @@
+#lang racket
```

Comment:
It would be great if the version could be inserted somewhere after the `#lang` instruction, so that the helper script `version-sync` will be able to automatically create the version file for this exercise.

It is discussed in greater detail in #131

Reply:
Sorry, there is actually no version for the canonical data since there is no canonical data for this exercise :sweat_smile:

Reply:
The lack of canonical data tripped me up at first too.